components/SectionTestnets.vue

```diff
@@ -540,6 +690,23 @@ export default {}
   background transparent !important
   color var(--gray-600) !important
 
+.list-title
+  margin-bottom var(--spacing-5)
+
+.list-item
+  position relative
+  &::after
+    content "↗"
```

Comment:
This should be on the anchor element so it is part of the link and inherits the color etc.